cdap-docs/integrations/source/partners/cloudera/ingesting.rst

```diff
@@ -99,59 +99,70 @@ are written in a custom format, they cannot be directly queried through Impala.
 you can create an ETL batch application that regularly reads
 stream events and writes those events into files on HDFS that can then be queried by Impala.
 
-To do this, write the following JSON to a config file::
+To do this, write the following JSON to a config file:
 
+.. parsed-literal::
+  :class: copyable copyable-text
+           
   {
+    "name": "streamToTPFS",
     "description": "Periodically reads stream data and writes it to a TimePartitionedFileSet",
+    "artifact": {
+      "name": "cdap-data-pipeline",
+      "scope": "system",
+      "version": "|release|"
+    },
     "config": {
-      "schedule": "*/10 * * * *",
-      "source": {
-        "name": "tradeStream",
-        "plugin": {
-          "name": "Stream",
-          "properties": {
-            "name": "trades",
-            "duration": "10m",
-            "format": "csv",
-            "schema": "{
-              \"type\":\"record\",
-              \"name\":\"purchase\",
-              \"fields\":[
-                {\"name\":\"ticker\",\"type\":\"string\"},
-                {\"name\":\"price\",\"type\":\"double\"},
-                {\"name\":\"trades\",\"type\":\"int\"}
-              ]
-            }",
-            "format.setting.delimiter":","
+      "schedule": "\*/10 \* \* \* \*",
```

Comment:
The escapes are for reST formatting. They don't appear in the [rendered version](http://builds.cask.co/artifact/CDAP-DQB66/shared/build-1/Docs-HTML/3.5.0-SNAPSHOT/en/integrations/partners/cloudera/ingesting.html). Otherwise, Sphinx thinks we want something in italic, which doesn't work very well. It's because this is a `parsed literal`, and so formatting is taken into account, at least as far as replacements go. Note that the backslashes get doubled in the schema.

See [the page](http://builds.cask.co/artifact/CDAP-DQB66/shared/build-1/Docs-HTML/3.5.0-SNAPSHOT/en/integrations/partners/cloudera/ingesting.html) for how it shows up. If you copy that JSON, it will work fine.